wb-codemod/package.json

```diff
@@ -1,10 +1,18 @@
 {
   "name": "@khanacademy/wb-codemod",
-  "version": "2.1.1",
   "license": "MIT",
+  "version": "2.1.1",
```

Comment:
Sorry, a bit nitpicky... all other published packages have these keys in this order: `license` then `version`.